connectors/sources/confluence.py

```diff
@@ -419,6 +468,31 @@ async def get_access_control(self):
                     user=user
                 )
 
+    async def get_access_control(self):
+        """Get access control documents for active Atlassian users.
+
+        This method fetches access control documents for active Atlassian users when document level security (DLS)
+        is enabled.
+        If DLS is enabled, the method fetches all users from the Confluence API, filters out active Atlassian users,
+        and fetches additional information for each active user using the fetch_user method. After gathering the user information,
+        it generates an access control document for each user using the user_access_control_doc method and yields the results.
+
+        Yields:
+            dict: An access control document for each active Atlassian user.
+        """
+        if not self._dls_enabled():
+            self._logger.warning("DLS is not enabled. Skipping")
+            return
+
+        self._logger.info("Fetching all users")
+        if self.confluence_client.data_source_type == CONFLUENCE_CLOUD:
+            users = self.get_user()
+        else:
+            users = self.get_user_for_server()
```

Comment:
So Jira Data Center and Jira Server share endpoints for permissions and permission concepts?

Reply:
Yes for fetching permissions both Jira Data Center and Jira Server  shares same Endpoints. There is just one dependency for both i.e. we need to install `Extender for Confluence` to execute the API's.

Reply:
What happens if this extender is not installed, but DLS is turned on?

Reply:
It will throw page not found error to us

Reply:
As of now it shows the same but let me update with custom message

Reply:
@akanshi-elastic have you added the human-readable error for the UI yet? I can't seem to find any changes including it.